scripts/fast-reboot

```diff
@@ -305,10 +306,11 @@ function reboot_pre_check()
         exit ${EXIT_FILE_SYSTEM_FULL}
     fi
 
-    # Make sure that the next image exists
-    if [[ ! -d ${IMAGE_PATH} ]]; then
-        debug "Next image ${NEXT_SONIC_IMAGE} doesn't exist ..."
-        exit ${EXIT_NEXT_IMAGE_NOT_EXISTS}
+    # Verify the next image by sonic_installer
+    local message=$(sonic_installer verify_next_image 2>&1)
+    if [ $? -ne 0 ]; then
+        debug "Failed to verify next image by running command: sonic_installer verify_next_image, result message: ${message}"
```

Comment:
No need to output the full command in the message. Suggest shortening message to:

```
"Failed to verify next image: ${message}"
```

Reply:
Fixed.